Project.toml

```diff
@@ -11,7 +11,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 Flux = "0.10.4, 0.11, 0.12"
-Lighthouse = "0.11"
+Lighthouse = "0.11.3"
```

Comment:
Is this now incompatible with Lighthouse 0.11.2?

Reply:
```suggestion
Lighthouse = "0.11"
```